packages/providers/package.json

```diff
@@ -18,8 +18,8 @@
   ],
   "scripts": {
     "update-schema": "get-graphql-schema http://localhost:4000/graphql > fuel-core-schema.graphql && prettier --write fuel-core-schema.graphql",
-    "build-operations": "pnpm graphql-codegen",
-    "prebuild": "pnpm build-operations",
+    "build:operations": "pnpm graphql-codegen",
+    "prebuild": "pnpm build:operations",
```

Comment:
Since we're here, could you kindly normalize the `update-schema` and re-sort? 😩 

```json
    "prebuild": "pnpm build:operations",
    "build": "tsup",
    "build:schema": "get-graphql-schema ....",
    "build:operations": "pnpm graphql-codegen",
    "postbuild": "tsx ../../scripts/postbuild.ts"
```

Reply:
Sure. On the topic of re-sorting, I've created issue #1289.